CONTRIBUTING.md

```diff
@@ -18,7 +18,7 @@ GitHub, clone, and develop on a branch. Steps:
 
 1. Fork the [project repository](https://github.com/scikit-learn/scikit-learn)
    by clicking on the 'Fork' button near the top right of the page. This creates
-   a copy of the code under your GitHub user account.
+   a copy of the code under your GitHub user account. For more details on how to fork a repository see [this guide.](https://help.github.com/articles/fork-a-repo/)
```

Comment:
Your lines are too long. Find an editor that breaks lines after a decent width say 80.